hack/bootstrap-cluster.sh

```diff
@@ -112,6 +112,26 @@ if ! kubectl get secret -n gitops gitops-postgresql-staging &>/dev/null; then
     --from-literal=postgresql-password=$(openssl rand -base64 20)
 fi
 
+echo
+echo "Setting secrets and config map for Quality Dashboard"
+if ! kubectl get namespace quality-dashboard &>/dev/null; then
+  kubectl create namespace quality-dashboard
+fi
+if ! kubectl get secret -n quality-dashboard quality-dashboard-secrets &>/dev/null; then
+  kubectl create secret generic quality-dashboard-secrets \
+    --namespace=quality-dashboard \
+    --from-literal=POSTGRES_USER=user \
+    --from-literal=POSTGRES_PASSWORD=$(openssl rand -base64 20) \
+    --from-literal=POSTGRESQL_DATABASE=dashboard-database \
+    --from-literal=github-token=token-for-read
+fi
+echo "Create backend URL for Quality Dashboard"
+if ! kubectl get configmap -n quality-dashboard quality-dashboard-configmap &>/dev/null; then
```

Comment:
This will not work on stage. This configmap needs to be defined in the repo & managed by gitops (this script will not be run against existing deployment - stage) - I guess we could hardcode the stage dashboard URL, in this repo and for the development/ephemeral cluster purposes we need to use the mechanism of "preview" mode - We could do the change to ephemeral cluster URL in `development` overlay.

Reply:
fixed by patch in kustomize.yaml for staging.